helpers.py

```diff
@@ -96,7 +96,10 @@ def rate_plot(output, var_name):
         color_discrete_sequence=["#56B4E9", "#E69F00"], 
         template="plotly_white")
     
-    fig.update_traces(hovertemplate=None)
+    if var_name != 'annual_net_income':
+        fig.update_traces(hovertemplate="%{y:.1%}")
```

Comment:
I don't understand this line and why it is different from the next one

Reply:
yeah the plotly hovertemplate is a bit cryptic (and the documentation is hopeless)  I'll add some comments